RosettaCore/src/main/java/com/hubspot/rosetta/RosettaBinder.java

```diff
@@ -15,6 +16,24 @@ public interface Callback {
     void bind(String key, Object value);
   }
 
+  public void bindList(ArrayNode node, String field, Consumer<Object> valueConsumer) {
```

Comment:
nit: call this `array` for clarity?
```suggestion
  public void bindList(ArrayNode array, String field, Consumer<Object> valueConsumer) {
```